server/src/services/server.service.ts

```diff
@@ -131,11 +132,26 @@ export class ServerService extends BaseService {
       usage.usageVideos = user.usageVideos;
       usage.quotaSizeInBytes = user.quotaSizeInBytes;
 
+      for (const path of user.encodedVideoPaths) {
+        if (path.trim() === '') {
+          continue;
+        }
+
+        try {
+          const stats = fs.statSync(path);
```

Comment:
This is terribly slow and will scale poorly as the number of videos increases, both in fetching every single path from the DB at once and in `stat`'ing them like this.